lls/main.c

```diff
@@ -430,6 +399,68 @@ lls_proc(void *arg)
 	return cleanup_lls();
 }
 
+static void
+fill_nd_rule(struct ipv6_acl_rule *rule, struct in6_addr *addr, int nd_type)
+{
+	uint32_t *ptr32 = (uint32_t *)addr;
+	int i;
+
+	RTE_VERIFY(nd_type == ND_NEIGHBOR_SOLICITATION ||
+		nd_type == ND_NEIGHBOR_ADVERTISEMENT);
+
+	rule->data.category_mask = 0x1;
+	rule->data.priority = 1;
+	/* Userdata is filled in in register_ipv6_acl(). */
+
+	rule->field[PROTO_FIELD_IPV6].value.u8 = IPPROTO_ICMPV6;
+	rule->field[PROTO_FIELD_IPV6].mask_range.u8 = 0xFF;
+
+	for (i = DST1_FIELD_IPV6; i <= DST4_FIELD_IPV6; i++) {
+		rule->field[i].value.u32 = rte_be_to_cpu_32(*ptr32);
+		rule->field[i].mask_range.u32 = 32;
```

Comment:
No, it's a configuration quirk. If the type of the field is `RTE_ACL_FIELD_TYPE_MASK`, then it represents a mask length.